### PR TITLE
update to the silence script

### PR DESCRIPTION
**Pull request by tobiasd**
Follow-up for #3948

### CODE REVIEW

File: util/global_community_silence.php [hunk 1]
```diff
@@ -36,11 +36,11 @@
 use Friendica\Database\DBM;
 use Friendica\Network\Probe;
 
-require_once("boot.php");
+require_once'"boot.php');
```
Comment (MrPetovan) [inline util/global_community_silence.php]: This is invalid. Please remove the double quote and the parenthesis.

File: util/global_community_silence.php [hunk 2]
```diff
@@ -36,11 +36,11 @@
...
 require_once('include/dba.php');
-require_once("include/text.php");
+require_once('include/text.php');
 $a = get_app();
-require_once ".htconfig.php";
+require_once('.htconfig.php');
```
Comment (MrPetovan) [inline util/global_community_silence.php]: Standards: Please remove the parentheses from language constructs.

File: util/global_community_silence.php (hunk 1 above)
Comment (tobiasd) [reply util/global_community_silence.php]: yea, thick air and I just realized what you wrote ;-)

File: util/global_community_silence.php [hunk 3]
```diff
@@ -36,11 +36,11 @@
...
 
-require_once'"boot.php');
-require_once('include/dba.php');
-require_once('include/text.php');
+require_once 'boot.php';
+require_once 'include/dba.php';
+require_once 'include/text.php';
 $a = get_app();
-require_once('.htconfig.php');
+require_once'.htconfig.php';
```
Comment (MrPetovan) [inline util/global_community_silence.php]: Not sure if a space is needed after require_once but we have been adding one so far.

**Pull request merged**